src/test/java/org/onebusaway/alexa/AuthedSpeechletTest.java

```diff
@@ -126,6 +126,11 @@ public void initializeAuthedSpeechlet() throws URISyntaxException {
         authedSpeechlet.setUserData(testUserData);
     }
 
+    @Before
+    public void resetFactoryObjects() {
+        session = Session.builder().withUser(testUser).withSessionId("test-session-id").build();
```

Comment:
Seems like it better emulates actual Alexa behavior.  Although I'm having a hard time coming up with a scenario where we would actually use the session ID.  I'm OK either way.